app/views/orgs/lti_configurations/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="remodal-bg">
+  <%= render 'organizations/organization_banner' %>
+
+  <div class="columns">
+    <div class="one-fourth column">
+      <%= render partial: 'organizations/side_menu' %>
+    </div>
+    <div class="three-fourths column">
+      <div class="site-content">
+        <div class="site-content-cap">
+          <h2 class="site-content-heading">Link classrom to your institution's LMS</h2>
+        </div>
+
+        <div class="site-content-body markdown-body">
+          <p>Connecting GitHub Classroom to your Learning Management System (LMS) will allow you to manage your students and submissions easier!</p>
+          <h3>What integrating Classroom with your LMS can do for you</h3>
+          <ul>
+            <li>Sync rosters from your institution's LMS with Classroom</li>
+            <li>Automatically create an assignment on Classroom when you create one on your LMS</li>
+            <li>Sync assignment submissions from Classroom to your LMS</li>
+          </ul>
+
+          <h3>How to link Classroom with your institution's LMS</h3>
+          <p>You or your system administrator will need to add a new application to
+          your Learning Management System with the following credentials generated by
+          GitHub Classroom.</p>
+          <ul>
+            <li>Consumer Key</li>
+            <li>Shared secret</li>
+            <li>Link to the LMS classroom page</li>
+          </ul>
+
+          <%= button_to "Connect to LMS", lti_configuration_path(current_organization), class: 'btn btn-primary' %>
```

Comment:
Hitting this error when I click this button in development: `No template found for Orgs::LtiConfigurationsController#create`

Reply:
This is expected since we don't have a view for this and the method in the controller is empty. This will be fixed in our next PR.